internal/repository/migrations/000001_create_application_input_claim_output_report_nodeconfig.up.sql

```diff
@@ -9,13 +9,18 @@ CREATE TYPE "ClaimStatus" AS ENUM ('PENDING', 'SUBMITTED', 'FINALIZED');
 
 CREATE TYPE "DefaultBlock" AS ENUM ('FINALIZED', 'LATEST', 'PENDING', 'SAFE');
 
+CREATE FUNCTION public.f_maxuint64()
+  RETURNS NUMERIC(20,0)
+  LANGUAGE sql IMMUTABLE PARALLEL SAFE AS
+'SELECT 18446744073709551615';
+
 CREATE TABLE "application"
 (
     "id" SERIAL,
```

Comment:
Maintaining it as "Serial" and not change to "Bigserial" is intended, right? As the number of applications at the same node should not be that big

Reply:
Yes, it's just as you said. No need to use `bigserial` here, unless we expect more than 2147483647 applications per node,